test/albumTest.js

```diff
@@ -109,13 +115,39 @@ describe('/albums/:id POST', () => {
         chai
           .request(server)
           .post('/albums/105')
+          .set(sessionManager.HEADER_NAME, loginRes.headers[sessionManager.HEADER_NAME])
           .catch(err => {
-            err.should.have.status(401);
+            err.should.have.status(404);
             err.response.should.be.json;
-            err.response.body.should.have.property('message');
+            err.response.body.should.have.property('error');
             done();
           });
       })
     );
   });
+
+  it('should return error for album with id 1 because the album has been already bought', done => {
+    User.create({
+      name: 'Franco',
+      surname: 'Coronel',
+      email: 'franco.coronel@wolox.com.ar',
+      password: 'passwordFC'
+    }).then(
+      successfullLogin().then(loginRes => {
+        request
+          .set(sessionManager.HEADER_NAME, loginRes.headers[sessionManager.HEADER_NAME])
+          .then(albumBought => {
+            chai
+              .request(server)
+              .post('/albums?id=1')
+              .catch(err => {
+                err.should.have.status(404);
```

Comment:
If the album has been already bought I would expect an error code 422. 404 means Not found, like the previous case where the album does not exists.